src/docs/releases/2.0.0.md

```diff
@@ -735,3 +735,7 @@ Many type commonly used by modules can be `sealed`, which improves runtime perfo
 
 !!! note
     Do not seal classes that are used to create shapes like view-models. Sealing these classes can break your code at runtime, as these classes need to be unsealed to allow for proxy creation.
+
+### Workflow Trimming
```

Comment:
```suggestion
#### Workflow Trimming
```

Reply:
It's not a sub-header of "Sealing Types" above. The parent of this is `## Change Logs`.